src/unicef_security/pipeline.py

```diff
@@ -17,3 +18,19 @@ def social_details(backend, details, response, *args, **kwargs):
 
 def get_username(strategy, details, backend, user=None, *args, **kwargs):
     return {'username': details.get('email')}
+
+
+def create_unicef_user(strategy, details, backend, user=None, *args, **kwargs):
+    """Overrides create_user, to create only UNICEF users"""
+    if user:
+        return {'is_new': False}
+
+    fields = dict((name, kwargs.get(name, details.get(name)))
+                  for name in backend.setting('USER_FIELDS', USER_FIELDS))
+    if not (fields and details.get('email', '').endswith('@unicef.org')):
```

Comment:
can we move the `endswith` string ("@unicef.org") into a setting?

Reply:
Done!
Do you see "@unicef.org" to change in the future?

Reply:
@domdinicola It allows one to easily override the setting in tests, so you're not required to use `@unicef.org` in the tests and can instead make use of `@example.com`